packages/core/src/sass/app/_common/layout/dashboard-header.scss

```diff
@@ -9,30 +9,48 @@
     top: 0;
     transition: opacity 0.3s;
     backface-visibility: hidden;
+    padding: 0 4rem;
+    justify-content: space-between;
+
+    > * {
+        align-items: center;
+        display: flex;
+    }
+
+    &--left {
+        > * {
+            cursor: pointer;
+        }
+    }
+}
+
+.logged-out-dashboard-header {
```

Comment:
Can we scope these states (line 27 and 43) under a `.dashboard-header` class? e.g.:

```SASS
.dashboard-header {
    &--logged-in {
        // Some logged in rules.
    } 
    &--logged-out {
        // Some logged out rules.
    } 
}
```

http://getbem.com/naming/ and search for `Modifier`